test.asyncapi-cli

```diff
@@ -0,0 +1 @@
+{"store":{}}
```

Comment:
Is this needed @Souvikns ?

Reply:
I think we can add it to the `gitignore`

Reply:
Done!